### DISCUSSION TITLE
Translations for wildwebdeveloper plugins

### DISCUSSION THREAD

**Discussion by RajasekharPat** [mickaelistria/eclipse-bluesky#1188] [Q&A]
Where can we find nl fragments for wildwebdeveloper plugins?

**Comment by mickaelistria** [comment]
Wild Web Developer per se doesn't maintain translations, although it uses code patterns that make it open for translations. Translations are usually provided by Eclipse Babel: https://eclipse.dev/babel/ ; please look there.
